karate-tests/src/test/java/patients/healthcareWorker/updatePatientDetails/replace.feature

```diff
@@ -81,19 +81,25 @@ Feature: Patch patient - Replace data
       * header Content-Type = "application/json-patch+json"
       * header If-Match = karate.response.header('etag')
       * path 'Patient', nhsNumber
+      * def mobileNumber = '0788548987'
       * request read('classpath:patients/requestDetails/add/emergencyContact.json')
       * method patch
       * status 200
       * def versionIdAftAdd = response.meta.versionId
```

Comment:
I think this is unused now, but speaking of which what's the reason for doing an extra GET instead of retrieving stuff like the version ID from the PATCH response?

Reply:
Removed unused variable. Given that parallel execution is utilized and P9's NHS numbers are shared between tests and teams, the majority of update-related tests perform a preliminary GET request before proceeding with the subsequent update operation. This approach is necessary to avoid version mismatch errors

Reply:
Thanks David for highlighting it, most of the PATCH- sandbox related tests uses GET request to get the original versionId